src/main/java/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor.java

```diff
@@ -312,4 +341,11 @@ public String getDisplayName() {
             return "Parallel test job execution";
         }
     }
+
+    static Predicate<FlowNode> stageNamePredicate(@Nonnull String stageName) {
+        // Arrays.asList in to prevent compiler warning due to unchecked generics/varargs whackiness.
+        return Predicates.and(Arrays.asList(new NodeStepTypePredicate("stage"),
```

Comment:
Safer to check the `StepDescriptor.id`.

Reply:
I prefer this for a possibly dumb reason: I don't have to write out the full package/class name here, and/or I don't have to have `pipeline-stage-step` as a dependency.